case-lib/lib.sh

```diff
@@ -252,6 +252,19 @@ func_lib_get_tplg_path()
     return 0
 }
 
+func_nocodec_mode()
```

Comment:
Can we dump the result at first and only use the variable later? Three is no need to check this mode multiple time

Reply:
> Can we dump the result at first and only use the variable later? 

This would be premature optimization. This command is only forking two additional and small processes per alsabat test, that's negligible.